In [2]:
import pandas as pd
import numpy as np
import math

## Context


It is assumed that the mean systolic blood pressure is μ = 120 mm Hg. 

In the Honolulu Heart Study, a sample of n = 100 people had an average 
systolic blood pressure of 130.1 mm Hg with a standard deviation of 21.21 mm Hg. 

Is the group significantly different (with respect to systolic blood pressure!) 
from the regular population?

Set up the hypothesis test.

Write down all the steps followed for setting up the test.

Calculate the test statistic by hand and also code it in Python. 

It should be 4.76190. 

### 1) Define the Null Hypthesis and the Alternative Hypothesis:


__Null Hypothesis:__
The mean systolic blood pressure is 120 mm Hg (μ = 120 mm Hg)



__Alternative Hypothesis:__
A sample from 100 peope had a mean systolic blood prssure of 130.1 mm Hg and standard deviation of 21.21 mm Hg (μ = 130.1 mm Hg; std = 21.21 mm Hg)

 ### 2) Since the alternative hypothesis wants to determine that the mean of the sample is different compared to the null hypothesis - this can be considered a two tail test

### 3) Set alpha as p = 0.05 

For a two tailed test with sample size 100 this would give the value **1.984**

### 4) Calculate the statistic

- we chose the t-test here because:
- we do not know the standard deviation from the population
- we have only 100 samples
- we assume that the samples are normally distributed

In [3]:
sample_mean = 130.1 
pop_mean = 120.0
sample_std = 21.21
n = 100

statistic = (sample_mean - pop_mean)/(sample_std/math.sqrt(n))
print('The observed statistic is: ', statistic)

the observed statistic is:  4.761904761904759


### 5) Conclusion

- given that the observed statistic is greater than alpha (4.762 > 1.984) we can reject the null nypothesis
- this result indicates that the probability of the observed statistic being drawn from the null hypothesis distribution (being observed in a world where the null hypothesis is true) is lower than 5% - and thus - as it is lower than our preset threshold we reject it